src/go/rpk/pkg/cli/redpanda/start.go

```diff
@@ -698,10 +700,10 @@ func check(
 			if action, exists := checkFailedActions[result.CheckerID]; exists {
 				action(&result)
 			}
-			msg := fmt.Sprintf("System check '%s' failed. Required: %v, Current %v",
-				result.Desc, result.Required, result.Current)
+			msg := fmt.Sprintf("System check '%s' failed. Required: %v, Current %v, Error: %v",
+				result.Desc, result.Required, result.Current, result.Err)
 			if result.Severity == tuners.Fatal {
-				return fmt.Errorf(msg)
+				return errors.New(msg)
```

Comment:
Just out of consistency with other newer parts of the code: we try to use `errors.New` if we will print just a message and `fmt.Errorf` if we are going to 'format' the message.

Since it's already formatted here, I changed that to errors.New.